index.html

```diff
@@ -8,41 +8,305 @@
+            /></a>
+            <a href="#"
+              ><img class="social-icons" src="assets/m.svg" alt="m"
+            /></a>
+            <a href="#"
+              ><img class="social-icons" src="assets/github.svg" alt="github"
+            /></a>
+            <a href="#"
+              ><img class="social-icons" src="assets/peace.svg" alt="peace"
+            /></a>
+          </article>
+        </aside>
+      </section>
+    </main>
+
+    <section class="grid">
+      <aside>
+        <article class="work-card-container">
+          <img src="assets/Work1.png" alt="first project" />
+          <h2>Tonic</h2>
+          <div class="subtitle-card-container">
+            <h4>CANOPY</h4>
+            <ul class="ul-flex-features">
+              <li>
+                <img
+                  class="icon-circle"
+                  src="assets/icon-circle.svg"
+                  alt="icon circle"
+                />
+              </li>
+              <li>Back End Dev</li>
+              <li>
+                <img
+                  class="icon-circle"
+                  src="assets/icon-circle.svg"
+                  alt="icon circle"
+                />
+              </li>
+              <li>2015</li>
+            </ul>
+          </div>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required
+          </p>
+          <div>
+            <ul class="ul-stack-flex">
+              <li class="li-style">HTML</li>
+              <li class="li-style">CSS</li>
+              <li class="li-style">JavaScript</li>
+            </ul>
+          </div>
+          <button class="btn-enable">See project</button>
+        </article>
+      </aside>
+
+      <aside>
+        <article class="work-card-container">
+          <img src="assets/Work2.png" alt="first project" />
+          <h2>Multi-Post Stories</h2>
+          <div class="subtitle-card-container">
+            <h4>CANOPY</h4>
+            <ul class="ul-flex-features">
+              <li>
+                <img
+                  class="icon-circle"
+                  src="assets/icon-circle.svg"
+                  alt="icon circle"
+                />
+              </li>
+              <li>Back End Dev</li>
+              <li>
+                <img
+                  class="icon-circle"
+                  src="assets/icon-circle.svg"
+                  alt="icon circle"
+                />
+              </li>
+              <li>2015</li>
+            </ul>
+          </div>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required
+          </p>
+          <div>
+            <ul class="ul-stack-flex">
+              <li class="li-style">HTML</li>
+              <li class="li-style">CSS</li>
+              <li class="li-style">JavaScript</li>
+            </ul>
+          </div>
+          <button class="btn-enable">See project</button>
```

Comment:
- [ ] **[OPTIONAL]** You've done an amazing job implementing all the suggestions provided by the previous reviewer! I have just one small optional suggestion **to further enhance the visual appeal of your project**. It would be fantastic if you could consider adding `hover, active, pressed, and disabled` effects to your buttons. These effects **can really elevate the overall look and user experience of your project**. Keep up the great work! 😉

   ![image](https://github.com/henrycadenaec/PortfolioFinishMobileVersion/assets/107952724/9a2bbbae-c5dd-4d8c-b440-34f328c5954a)

Reply:
Thank you very much @taldr27